src/main/java/com/schibsted/spt/data/jslt/impl/BuiltinFunctions.java

```diff
@@ -285,6 +289,30 @@ else if (!divisor.isNumber())
     }
   }
 
+  // ===== HASH-INT
+
+  public static class HashInt extends AbstractFunction {
+
+    private static ObjectMapper mapper = new ObjectMapper();
+    private static ObjectWriter writer = mapper.writerWithDefaultPrettyPrinter();
+
+    public HashInt() {
+      super("hash-int", 1, 1);
+    }
+
+    public JsonNode call(JsonNode input, JsonNode[] arguments) {
+      JsonNode node = arguments[0];
+      if (node.isNull())
+        return NullNode.instance;
+      try {
+        String jsonString = writer.writeValueAsString(node);
+        return new IntNode(jsonString.hashCode());
+      } catch (JsonProcessingException e) {
+        return NullNode.instance;
```

Comment:
We shouldn't silently fail in this case. Instead, the transform should crash and burn. Debugging this kind of thing can be very, very difficult otherwise.

Reply:
You're right, I'll throw `throw new JsltException("hash-int: can't process json" + e);` instead